src/ig/IGExporter.ts

```diff
@@ -659,22 +694,38 @@ export class IGExporter {
     const menuXMLOutputPath = path.join(igPath, 'input', 'includes', 'menu.xml');
 
     // If user provided menu file in input/includes and no config, copy over the file.
-    if (existsSync(menuXMLDefaultPath) && !this.config.menu) {
+    if (existsSync(menuXMLDefaultPath) && !this.config.menu && this.shouldCopyFiles) {
       this.copyWithWarningText(menuXMLDefaultPath, menuXMLOutputPath);
       return;
     }
 
     // If user provided file and config, log a warning but prefer the config.
     if (existsSync(menuXMLDefaultPath) && this.config.menu) {
-      logger.warn(
-        `Found both a "menu" property in ${this.configName} and a menu.xml file at ig-data${path.sep}input${path.sep}includes${path.sep}menu.xml. ` +
+      const filePathString = `${path.basename(this.igDataPath)}${path.sep}input${path.sep}includes${
+        path.sep
+      }menu.xml`;
```

Comment:
I've noticed this pattern w/ `${path.sep}` a few times now.  You know about `path.join`, right?  E.g., the above could be:
```
const filePathString = path.join(path.basename(this.igDataPath), 'input', 'includes', 'menu.xml');
```

Reply:
Oops this was definitely a tunnel vision issue, and I was too focused on what used to be there. I can go back through and use `path.join`.